SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -48,6 +50,18 @@ FString FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(const FString& A
 	return FPaths::ConvertRelativePathToFull(FPaths::Combine(PluginDir, AppendPath));
 }
 
+void FSpatialGDKServicesModule::DeleteSchemaDatabase()
+{
+	const FString SchemaDatabasePackagePath = TEXT("/Game/Spatial/SchemaDatabase");
+	const FString SchemaDatabaseAssetPath = FString::Printf(TEXT("%s.SchemaDatabase"), *SchemaDatabasePackagePath);
+	const FString SchemaDatabaseFileName = FPackageName::LongPackageNameToFilename(SchemaDatabasePackagePath, FPackageName::GetAssetPackageExtension());
+
+	if (!FPlatformFileManager::Get().GetPlatformFile().DeleteFile(*SchemaDatabaseFileName))
```

Comment:
What if it's read only?
An error messages should output the path of the asset also.

Reply:
https://github.com/spatialos/UnrealGDK/blob/198f371f4118e547e3c51d05677f93bcd43ae692/SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp#L544 checks source access control, should use this behaviour.

Reply:
TryLoadingExistingSchemaDatabase() looks like it's written specifically for generating schema; error messages say "Schema generation failed", plus it actually loads the schema database and sometimes calls ClearGeneratedSchema.